langgraph_bot/tools/tools.py

```
@@ -122,7 +120,7 @@ def pdfreader_tool(pdf_list: List[Dict]) -> List:
             loader = UnstructuredPDFLoader(pdf_path)
             docs = loader.load()
             all_docs.extend(docs)
-        except (requests.RequestException, Exception) as e:
+        except (requests.RequestException, Exception):
             # Log error or handle gracefully - skip this PDF and continue
             pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)